jest.config.json

```diff
@@ -12,7 +12,6 @@
   "testMatch": ["**/?(*.)+(spec|test).[jt]s?(x)"],
   "testPathIgnorePatterns": ["node_modules", "lib", "esm"],
   "moduleDirectories": ["src", "node_modules"],
-  "collectCoverage": true,
```

Comment:
Comment: I understand the need to not collect the coverage report when running in CI mode but I wonder if we can add something to conditionally opt-in/out when in CI mode as we can set an `env` in the workflows. I think it's a better approach than adding a new `coverage` script but I'm okay with it if the above isn't possible

```
const isCI = process.env.CI === 'true';

...

collectCoverage: !isCI